src/memory_dump/debug.h

```diff
@@ -0,0 +1,6 @@
+#ifdef _DEBUG
```

Comment:
Please  add CERE header licence file to all .h and .c (even small ones such as debug.h)